dll/cpl/sysdm/lang/ro-RO.rc

```diff
@@ -34,236 +34,236 @@ END
-    GROUPBOX "Este permis să fie folosit", -1, 8, 77, 185, 48
+    PUSHBUTTON "&Răsfoire", IDC_USERPROFILE_COPY_BROWSE, 15, 41, 50, 14
+    GROUPBOX "Acceptat pentru utilizare", -1, 8, 77, 185, 48
     LTEXT "", IDC_USERPROFILE_COPY_USER, 15, 91, 167, 8
-    PUSHBUTTON "S&chimbare", IDC_USERPROFILE_COPY_CHANGE, 15, 105, 50, 14
-    DEFPUSHBUTTON "Confirmă", IDOK, 204, 13, 50, 14
-    PUSHBUTTON "Anulează", IDCANCEL, 204, 30, 50, 14
+    PUSHBUTTON "&Modificare", IDC_USERPROFILE_COPY_CHANGE, 15, 105, 50, 14
+    DEFPUSHBUTTON "OK", IDOK, 204, 13, 50, 14
+    PUSHBUTTON "Revocare", IDCANCEL, 204, 30, 50, 14
 END
 
 IDD_STARTUPRECOVERY DIALOGEX 6, 18, 252, 280
 STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CONTEXTHELP | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
-CAPTION "Pornire și Recuperare"
+CAPTION "Pornire şi recuperare"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    GROUPBOX "Pornirea sistemului", IDC_STATIC, 7, 12, 238, 95
-    LTEXT "Sistemul de operare implicit:", IDC_STATIC, 14, 26, 100, 8
+    GROUPBOX "Pornire sistem", IDC_STATIC, 7, 12, 238, 95
+    LTEXT "Sistem de &operare implicit:", IDC_STATIC, 14, 26, 100, 8
     COMBOBOX IDC_STRECOSCOMBO, 14, 37, 224, 46, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
-    AUTOCHECKBOX "&Perioada de afișare a sistemelor de operare:", IDC_STRECLIST, 14, 56, 160, 8
+    AUTOCHECKBOX "&Timp de afişare a sistemelor de operare:", IDC_STRECLIST, 14, 56, 160, 8
     EDITTEXT IDC_STRRECLISTEDIT, 178, 54, 30, 12, ES_NUMBER
     CONTROL "", IDC_STRRECLISTUPDWN, "msctls_updown32", UDS_WRAP | UDS_SETBUDDYINT |
             UDS_ALIGNRIGHT | UDS_AUTOBUDDY | UDS_ARROWKEYS | UDS_NOTHOUSANDS | WS_CHILD | WS_VISIBLE, 0, 0, 8, 13
     LTEXT "secunde", IDC_STATIC, 209, 56, 30, 8
-    AUTOCHECKBOX "Perioada de afișare a &opțiunilor de recuperare:", IDC_STRRECREC, 14, 70, 163, 8
+    AUTOCHECKBOX "Timp pentru &afişarea a opţiunilor de recuperare:", IDC_STRRECREC, 14, 70, 163, 8
     EDITTEXT IDC_STRRECRECEDIT, 178, 68, 30, 12, ES_NUMBER
     CONTROL "", IDC_STRRECRECUPDWN, "msctls_updown32", UDS_WRAP | UDS_SETBUDDYINT |
             UDS_ALIGNRIGHT | UDS_AUTOBUDDY | UDS_ARROWKEYS | UDS_NOTHOUSANDS | WS_CHILD | WS_VISIBLE, 0, 0, 8, 13
     LTEXT "secunde", IDC_STATIC, 209, 70, 30, 8
-    LTEXT "Puteți alege editarea manuală a opțiunilor de pornire:", IDC_STATIC, 14, 89, 187, 8
-    PUSHBUTTON "&Editare…", IDC_STRRECEDIT, 188, 87, 50, 14
-    GROUPBOX "Căderea sistemului", IDC_STATIC, 7, 111, 238, 140
-    AUTOCHECKBOX "&Consemnează evenimentul în jurnalul de sistem", IDC_STRRECWRITEEVENT, 14, 124, 178, 10
-    AUTOCHECKBOX "T&ransmite alertă administrativă", IDC_STRRECSENDALERT, 14, 138, 148, 10
-    AUTOCHECKBOX "Repornește &automat sistemul", IDC_STRRECRESTART, 14, 152, 145, 10
-    GROUPBOX "Scriere informații de depanare", IDC_STATIC, 12, 167, 227, 76
+    LTEXT "Pentru a edita fişierul cu opţiuni de pornire, faceţi clic pe Editare.", IDC_STATIC, 14, 89, 187, 8
```

Comment:
This line really ends with a `.` and not with a `:` ?

Reply:
Yes

![2024-05-05 17_57_23-Resource Hacker - sysdm cpl mui](https://github.com/reactos/reactos/assets/113553683/fe83a699-595e-4e7d-9ec9-86b31ba617d3)